py2-numpy-toolfile.spec

```diff
@@ -21,6 +21,17 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/py2-numpy.xml
 </tool>
 EOF_TOOLFILE
 
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/py2-numpy-c-api.xml
+<tool name="py2-numpy-c-api" version="@TOOL_VERSION@">
+  <client>
+    <environment name="PY2_NUMPY_C_API_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INCLUDE" default="$PY2_NUMPY_C_API_BASE/lib/python@PYTHONV@/site-packages/numpy-@PY2_NUMPY_REAL_VERSION@-py@PYTHONV@-linux-@OSARCH@.egg/numpy/core/include"/>
```

Comment:
@davidlt , wouldn't it be cleaner if we create $PY2_NUMPY_ROOT/include symlink pointing to $PY2_NUMPY_ROOT/lib/python@PYTHONV@/site-packages/numpy-%realversion-py@PYTHONV@-linux-@OSARCH@.egg/numpy/core/include?